src/test/java/SafroN/Tests/ChangeLocationTestNegative.java

```diff
@@ -0,0 +1,26 @@
+package SafroN.Tests;
+import BasePageAndTestInit.TestInit;
+import SafroN.Pages.HotlineHeaderViewElements;
+import SafroN.Pages.HotlineLocationViewElements;
+import SafroN.Pages.HotlineLocationViewHelper;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ChangeLocationTestNegative extends TestInit {
+    //enter the correct location
+    @Test
+    public void test2(){
+        openUrl("https://hotline.ua/");
+        HotlineHeaderViewElements hotlineHeaderViewElements = new HotlineHeaderViewElements(driver);
+        hotlineHeaderViewElements.locationBtn().click();
+
+        HotlineLocationViewHelper hotlineLocationViewHelper = new HotlineLocationViewHelper(driver);
+        hotlineLocationViewHelper.search("SafroN");
+
+        HotlineLocationViewElements hotlineLocationViewElements = new HotlineLocationViewElements(driver);
+
+        Assert.assertTrue(hotlineLocationViewElements.exceptionMessage().isDisplayed());
+
```

Comment:
1. Delete comment (line 11)
1. Rename test1() method, please
2. Group objects of the classes together at the top of the test1() method
3. Remove empty lines between steps (including openUrl() method)
4. Remove empty line (line 24)